cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/common/MeasureEvaluator.java

```diff
@@ -184,94 +189,91 @@ protected void evaluateProportion(
         // add resources
         // add subject
 
+        // Evaluate Population Expressions
         initialPopulation = evaluatePopulationMembership(subjectType, subjectId, initialPopulation, evaluationResult);
-
-        if (initialPopulation.getSubjects().contains(subjectId)) {
-            // Evaluate Population Expressions
-            denominator = evaluatePopulationMembership(subjectType, subjectId, denominator, evaluationResult);
-            numerator = evaluatePopulationMembership(subjectType, subjectId, numerator, evaluationResult);
-            if (applyScoring) {
-                // remove denominator values not in IP
-                denominator.getResources().retainAll(initialPopulation.getResources());
-                denominator.getSubjects().retainAll(initialPopulation.getSubjects());
-                // remove numerator values if not in Denominator
-                numerator.getSubjects().retainAll(denominator.getSubjects());
-                numerator.getResources().retainAll(denominator.getResources());
+        denominator = evaluatePopulationMembership(subjectType, subjectId, denominator, evaluationResult);
```

Comment:
Question:  why did you remove the conditional logic for individiualpopulations.getsubjects().contains(subjectid) ?